src/mbgl/renderer/layers/render_line_layer.hpp

```diff
@@ -8,18 +8,37 @@
 #include <mbgl/layout/pattern_layout.hpp>
 #include <mbgl/gfx/texture.hpp>
 
+#include <optional>
+#include <memory>
+
 namespace mbgl {
 
 class LineProgram;
 class LineGradientProgram;
 class LineSDFProgram;
 class LinePatternProgram;
 
+namespace gfx {
```

Comment:
Wrap these in `#if MLN_DRAWABLE_RENDERER`, keeping forward decls out of the legacy path is helpful for catching other places we may forget to exclude code.